REFERENCE.md

```diff
@@ -760,6 +782,89 @@ Number of seconds to wait for Deployment and StatefulSet rollouts to complete.
 
 ## Plans
 
+### <a name="pam_toolsinstall_chart"></a>`pam_tools::install_chart`
+
+Install a helm chart.
+
+Optionally adds a helm *repository_uri* before installing.
+
+Optionally waits for rollout of the installed chart based on a
+*part_of* selector.
+
+#### Parameters
+
+The following parameters are available in the `pam_tools::install_chart` plan:
+
+* [`targets`](#targets)
+* [`chart_name`](#chart_name)
+* [`release`](#release)
+* [`repository_uri`](#repository_uri)
```

Comment:
Clicking this seems to take you to the `repository_uri` under "helm_add_repository" instead of "pam_tools::install_chart".

Reply:
Yes, looks like a puppet-strings bug.

Reply:
Sorry, I should clarify; REFERENCE.md is generated by rake strings:generate:reference which is a puppet-strings rake helper. If I manually edit for these issues, it will come into conflict or be reset the next time REFERENCE gets auto generated.

Reply:
Looks like puppet-strings is using github issues; I'll open a bug there.

Reply:
puppetlabs/puppet-strings#294